go.mod

```diff
@@ -183,3 +281,20 @@ replace (
 
 // Plugin kn-plugin-event from midstream
 replace knative.dev/kn-plugin-event => github.com/openshift-knative/kn-plugin-event v0.32.1-0.20220601003552-7c29d149ba16
+
+// Plugin kn-plugin-func from midstream
```

Comment:
There's plenty of alignment of `kn` X `func` modules. All seems to be building now. 🤞